CHANGELOG.md

```diff
@@ -7,8 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+- Fixed a bug in `repos().issues().delete_reaction()` which didn't account for HTTP 204 cases.
```

Comment:
Can you not include CHANGELOG.md changes, and instead include it in your commit? it will be automatically added to the changelog by release-plz.

Reply:
Sure! Done.